iacrcc/iacrcc.cls

```diff
@@ -495,19 +488,27 @@ xscale=\@OrigHeightRecip*\@curXheight,transform shape]
           Anonymous Submission
         \else
           \setcounter{footnote}{0}%
-          \setcounter{IACR@author@cnt}{1}%
+          \newcounter{IACR@cnt}
+          \setcounter{IACR@cnt}{1}%
+          % Make sure authors are separated by a comma except the last one where we use " and "
           \def\and{%
-            \stepcounter{IACR@author@cnt}%
-            \ifnum\theIACR@author@cnt=\IACR@author@last\unskip\space and \ignorespaces\else\unskip, \ignorespaces\fi}
-          \@author\if@IACR@autoinst\inst{\theIACR@author@cnt}\fi
+            \stepcounter{IACR@cnt}%
+            \ifnum\theIACR@cnt=\theIACR@author@cnt\unskip\space and \ignorespaces\else\unskip, \ignorespaces\fi%
+          }%
+          % Display the list of authors
+          % Note that affiliations and footnotes are picked up within 
+          % the definition of \@author
+          \@author%
           \vskip 1em\par
           \small
-          \setcounter{IACR@author@cnt}{1}%        
+          \setcounter{IACR@cnt}{1}%        
```

Comment:
This is a "generic" counter used temporarily in some of our loops.

Reply:
ah now I understand when I looked at the whole file. I think it would be more clear if we called this IACR@tmp@cnt or IACR@index. I tend to use the word "index" to mean the index in a loop, whereas I use "count" to actually count something.